modules/azure_vnet_injected_databricks_workspace/versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 3.10.0"
```

Comment:
should be consistent with the `versions.tf` of the `spoke_vnet` module

Reply:
> @rportilla-databricks which module do we want to publish, the azure_data_exfil, or the spoke_vnet & vnet_injected?

@nkvuong , I think the code has been modified since this comment but we want to publish 2 modules (spoke_vnet and the azure_vnet_injected_databricks_workspace module). Let me know your thoughts on the content before we merge this back to main. Thanks for the comments so far!

Reply:
makes sense to me - just those 2 minor comments I made then